database/nosql/mongodb/egg-mongo-d-b6.json

```diff
@@ -15,7 +15,7 @@
     "file_denylist": [],
     "startup": "mongod --fork --dbpath \/home\/container\/mongodb\/ --port ${SERVER_PORT} --bind_ip 0.0.0.0 --logpath \/home\/container\/logs\/mongo.log -f \/home\/container\/mongod.conf; until nc -z -v -w5 127.0.0.1 ${SERVER_PORT}; do echo 'Waiting for mongodb connection...'; sleep 5; done; mongosh --username ${MONGO_USER} --password ${MONGO_USER_PASS} --host 127.0.0.1:${SERVER_PORT} && mongosh --eval \"db.getSiblingDB('admin').shutdownServer()\" 127.0.0.1:${SERVER_PORT}",
     "config": {
-        "files": "{}",
```

Comment:
Pleas do not edit the files manually. This can cause issues if you didn't edit it correctly.

Reply:
I'm not sure I understand what you mean, I modified this line by changing the egg configuration from my panel, then exported the egg once the configuration was working to get the version currently present in the .json file.

Reply:
If you changed more in your egg than that You can also pick and choose changes to PR.

Reply:
Okay. Since I just changed the files property, I guess I just need to commit something to change the exported_at property